src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs

```diff
@@ -68,6 +73,14 @@ public override MetadataItem DefaultVisit(ISymbol symbol)
             item.Source = VisitorHelper.GetSourceDetail(symbol);
             var assemblyName = symbol.ContainingAssembly?.Name;
             item.AssemblyNameList = string.IsNullOrEmpty(assemblyName) ? null : new List<string> { assemblyName };
+#if NET451
+            var name = Path.GetFileName(ProjectPath);
+            if (name.Equals("project.json", StringComparison.OrdinalIgnoreCase)) {
+                var project = ProjectReader.GetProject(ProjectPath);
+                var packageName = $"{assemblyName}({project.Version.ToString()})";
+                item.PackageNameList = string.IsNullOrEmpty(packageName) ? null : new List<string> { packageName };
```

Comment:
packageName can't be NullOrEmpty here, needn't check. checking null for ProjectPath or project or project.Version?